setup.py

```diff
@@ -1,48 +1,98 @@
 from __future__ import absolute_import, division, print_function
 
-from setuptools import setup, find_packages, Extension
+from distutils.command.sdist import sdist as _sdist
+import glob
 import os
 
```

Comment:
@tjtg Not you're doing, but since we're in this space, let's take the opportunity to do some tidying up... hope you okay with that? :wink: 

So, nuke this blank line, thanks